tests/test_analysis/test_jit_analysis.py

```diff
@@ -0,0 +1,800 @@
+            analyzer_copy._enable_warn_uncalled_mods,
+        )
+        self.assertEqual(analyzer._warn_trace, analyzer_copy._warn_trace)
+
+        # Changing copy does not change original
+        analyzer_copy.unsupported_ops_warnings(enabled=True)
+        self.assertNotEqual(
+            analyzer._enable_warn_unsupported_ops,
+            analyzer_copy._enable_warn_unsupported_ops,
+        )
+
+        # Copy with new model and inputs
+        new_model = NonForwardNet()
+        bs = 5
+        new_inputs = (torch.randn((bs, *new_model.input_size)), )
+        analyzer_new = analyzer.copy(
+            new_model=new_model, new_inputs=new_inputs)
+
+        non_forward_flops = new_model.fc_flops + new_model.submod.fc_flops
+
+        # Total is correct for new model and inputs
+        self.assertEqual(analyzer_new.total(), non_forward_flops * bs)
+
+        # Original is unaffected
+        self.assertEqual(analyzer.total(), repeated_net_flops)
+
+        # Settings match
+        self.assertEqual(
+            analyzer._enable_warn_unsupported_ops,
+            analyzer_new._enable_warn_unsupported_ops,
+        )
+        self.assertEqual(analyzer._warn_trace, analyzer_new._warn_trace)
+
+    def test_disable_warnings(self) -> None:
+        """Tests .unsupported_ops_warnings(...) and .tracer_warnings(...)"""
+        model = TraceWarningNet()
+        inputs = (torch.randn((1, *model.input_size)), )
+        analyzer = FlopAnalyzer(model=model, inputs=inputs)
+
+        # Tracer warnings
+        analyzer.tracer_warnings(mode='all')
+        analyzer._stats = None  # Manually clear cache so trace is rerun
+        self.assertWarns(torch.jit.TracerWarning, analyzer.total)
+        analyzer._stats = None  # Manually clear cache so trace is rerun
+        self.assertWarns(RuntimeWarning, analyzer.total)
+
+        analyzer.tracer_warnings(mode='none')
+        analyzer._stats = None  # Manually clear cache so trace is rerun
+        with warnings.catch_warnings(record=True) as w:
+            warnings.simplefilter('always')
+            _ = analyzer.total()
+            if w:
+                warning_types = [s.category for s in w]
+                self.assertFalse(torch.jit.TracerWarning in warning_types)
+                self.assertFalse(RuntimeWarning in warning_types)
+
+        analyzer.tracer_warnings(mode='no_tracer_warning')
+        analyzer._stats = None  # Manually clear cache so trace is rerun
+        self.assertWarns(RuntimeWarning, analyzer.total)
+        analyzer._stats = None  # Manually clear cache so trace is rerun
+        with warnings.catch_warnings(record=True) as w:
+            warnings.simplefilter('always')
+            _ = analyzer.total()
+            if w:
+                warning_types = [s.category for s in w]
+                self.assertFalse(torch.jit.TracerWarning in warning_types)
+
+        # Unsupported ops and uncalled modules warnings
+
+        logger = logging.getLogger()
```

Comment:
Why use `logging.getLogger` here, and use `MMLogger.get_current_instance()` in line 775